packages/mock/src/faker/resolvers/value.ts

```diff
@@ -101,8 +105,24 @@ export const resolveMockValue = ({
       },
       imports,
       existingReferencedProperties,
+      functions,
       allowOverride,
     });
+    if (newSchema.allOf) {
```

Comment:
Not only can objects be separated using `oneOf`, but also objects can now be separated using `allOf`? ?

Reply:
My mistake, I did this backwards. I have now changed this to look at the `combine.separator === 'oneOf` instead, which is much better.